content/en/docs/v3.5/quickstart.md

```diff
@@ -1,61 +1,36 @@
 ---
 title: Quickstart
 weight: 900
-description: Get etcd up and running in less than 5 minutes!
+description: Run etcd Locally Under 5 minutes!
 ---
 
-Follow these instructions to locally install, run, and test a single-member
-cluster of etcd:
+This guide walks through installing and running a single-node etcd cluster locally using pre-built binaries - no containers or cloud set up required.
 
- 1. Install etcd from pre-built binaries or from source. For details, see
-    [Install][].
+## Prerequisites
 
-    {{% alert color="warning" %}}**Important**: Ensure that you perform the last
-    step of the installation instructions to verify that `etcd` is in your path.
-    {{% /alert %}}
+Make sure you're on a supported platform before getting started:
 
- 2. Launch `etcd`:
+- [Supported platforms architecture](/docs/v3.5/op-guide/supported-platform)
+- Tools installed:
+  - `curl` and `tar`
+  - Internet connection to download the latest etcd [release](https://github.com/etcd-io/etcd/releases/).
+  - `sudo` access to install binaries.
 
-    ```console
-    $ etcd
-    {"level":"info","ts":"2021-09-17T09:19:32.783-0400","caller":"etcdmain/etcd.go:72","msg":... }
-    ⋮
-    ```
+## Installation
 
-    {{% alert color="info" %}}**Note**: The output produced by `etcd` are
-    [logs](../op-guide/configuration/#logging) &mdash; info-level logs can
-    be ignored. {{% /alert %}}
-
- 3. From **another terminal**, use `etcdctl` to set a key:
-
-    ```console
-    $ etcdctl put greeting "Hello, etcd"
-    OK
-    ```
-
- 4. From the same terminal, retrieve the key:
-
-    ```console
-    $ etcdctl get greeting
-    greeting
-    Hello, etcd
-    ```
+To install etcd, please see the [installation guide](docs/v3.5/install/).
 
 ## What's next?
 
 Learn about more ways to configure and use etcd from the following pages:
 
-- Explore the gRPC [API][].
-- Set up a [multi-machine cluster][clustering].
-- Learn how to [configure][] etcd.
-- Find [language bindings and tools][integrations].
-- Use TLS to [secure an etcd cluster][security].
-- [Tune etcd][tuning].
-
-[api]: /docs/{{< param version >}}/learning/api
-[clustering]: /docs/{{< param version >}}/op-guide/clustering
-[configure]: /docs/{{< param version >}}/op-guide/configuration
-[integrations]: /docs/{{< param version >}}/integrations
-[security]: /docs/{{< param version >}}/op-guide/security
-[tuning]: /docs/{{< param version >}}/tuning
-[Install]: ../install/
+- If you are a developer:
```

Comment:
Please keep these consistent. 
i.e. "- If you are a developer:" vs "If you are an operator or admin:".